shopcart-backend/src/main/java/com/shopcart/backend/controller/ProductRestController.java

```diff
@@ -28,14 +28,18 @@ public ProductRestController(ProductRepository productRepository, ReviewReposito
     public List<Product> getAllProducts(
     	@RequestParam("name") Optional<String> name,
     	@RequestParam("brand") Optional<String> brand,
-    	@RequestParam("category") Optional<String> category)
+    	@RequestParam("category") Optional<String> category,
+    	@RequestParam("num") Optional<Integer> num)
```

Comment:
These are good. Maybe we should look into API pagination instead of item limits, anyways that's a future issue. Good work.